frameworks/Java/javalin/build.gradle

```diff
@@ -0,0 +1,44 @@
+apply plugin: 'java'
+apply plugin: 'eclipse'
+apply plugin: 'application'
+
+group 'com.lostsys.test.javalin'
+version '1.0-SNAPSHOT'
+
+mainClassName = 'com.lostsys.test.javalin.Bench'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile 'io.javalin:javalin:2.3.0'
+    compile "com.fasterxml.jackson.core:jackson-databind:2.9.6"
+    compile "org.slf4j:slf4j-simple:1.7.25"
+}
+
+/* ****************** */
+/*    Application     */
+/* ****************** */
+run {
+    if(project.hasProperty('args')){
+        args project.args.split(',')
+    }
+    jvmArgs = ['-XX:+UseParallelGC','-XX:+AggressiveOpts']//['-XX:+UseNUMA','-XX:+UseParallelGC','-XX:+AggressiveOpts']
```

Comment:
Did you mean to leave `-XX:+UseNUMA` commented out?  I haven't measured its performance impact for any framework in the Citrine environment, but if you're going for consistency with other frameworks, I suggest leaving it enabled.

Reply:
Added them. This is my first time working with Java. I don't understand much of what's going on with the arguments. I just frankenstein'd from implementations of other Java frameworks until it ran.

Reply:
These JVM arguments aren't necessary, but other people maintaining the Java implementations in this project have added them in an attempt to improve the performance results.  Here's what I know about each one:

* `+UseParallelGC`: Use Parallel GC instead of the default garbage collector.  Parallel GC performs better on throughput (which is the primary performance metric we use in TFB) than other collectors, but worse in other metrics such as worst-case latency.  We generally don't use Parallel GC in real-world projects, but it probably has a measurable benefit to requests/s in TFB.
* `+UseNUMA`: Enable optimizations that only apply for [NUMA](https://en.wikipedia.org/wiki/Non-uniform_memory_access) architecture, which the Citrine environment has.  As I understand it, this flag only does anything when Parallel GC is enabled.
* `+AggressiveOpts`: Enable a grab-bag of various optimizations with questionable benefits.  I believe this flag was deprecated in later versions of Java in favor of using more specific flags.

At some point I would like to test the impact of each of these flags in the Citrine environment on a few different representative Java frameworks, halting the continuous benchmarks while we run these tests manually.  Then we can understand once and for all how they affect the results.  Until then, using the same JVM arguments as other frameworks seems fine.  Omitting all of these flags would also be fine.